modules/oe_webtools_etrans/src/Plugin/Block/UnifiedEtransBlock.php

```diff
@@ -306,4 +311,27 @@ protected function preparesWtEtransJson(string $translation_from, string $placeh
     return $json;
   }
 
+  /**
+   * Attempt to map langcode to ISO 639-2.
+   *
+   * @param string $langcode
+   *   The drupal langcode.
+   *
+   * @return string
+   *   The ISO 639-2 langcode format.
+   */
+  protected function mapLangcodeToIso(string $langcode): string {
+    // In Drupal, langcode are often in ISO 639-2 format by default.
```

Comment:
```suggestion
    // In Drupal, langcode are often in ISO 639-1 format by default.
```
ISO 639-1 -> two-letters codes
ISO 639-2 -> three-letters codes

Reply:
Correct. We can commit this.